docs/types.rst

```diff
@@ -857,13 +859,31 @@ or create a new memory array and copy every element.
         }
     }
 
-.. index:: ! array;literals, !inline;arrays
+.. index:: ! array;literals, ! inline;arrays
 
-Array Literals / Inline Arrays
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Array Literals
+^^^^^^^^^^^^^^
 
-Array literals are arrays that are written as an expression and are not
-assigned to a variable right away.
+Array literals (also called inline arrays) are arrays that are written as an
+expression and are not assigned to a variable right away.
+
+::
+
+  pragma solidity >=0.4.16 <0.6.0;
+
+  contract C {
```

Comment:
This is the correct example, but it's the same as the one some lines further down...

Reply:
OK, I wanted to have a syntax example, even if it's really basic, and somewhat unrealistic. Hopefully this is OK.